sympy/core/function.py

```diff
@@ -718,7 +718,7 @@ def _sage_(self):
         import sage.all as sage
         fname = str(self.func)
         args = [arg._sage_() for arg in self.args]
-        func = sage.function(fname, *args)
+        func = sage.function(fname)(*args)
```

Comment:
No idea. It was giving a deprecation warning. I think we should only really bother with the latest version anyway, because the main idea is to work with Sage once it includes the latest version of SymPy inside of it.